person.rb

```diff
@@ -0,0 +1,38 @@
+# creating a Person class
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1000..9999)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  # define getter for @name
+  attr_accessor :name
+
+  # define setter for @name
+
+  # define getter for @age
+  attr_accessor :age
+
+  # define getter for age
+
+  def can_use_services?
+    if (age >= 18) || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] To follow best practices and clean code. Kindly refactor this method to use the `def of_age?` method that you created as private. In this way you save code and make it easy to read